FileManager+SFLoggingKit.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FileManager+SFLoggingKit.swift
+//  SFLoggingKit
+//
+//  Created by Aleksandar Sergeev Petrov on 23.11.21.
+//
+
+import Foundation
+
+extension FileManager {
+
+    func ensureFileExists(_ filePath: String) throws {
```

Comment:
[optional]

If we want to expose this I believe it should be public.

Reply:
we don't :)

internal use only